txengine-deploy/src/handle-certificate.js

```diff
@@ -0,0 +1,108 @@
+const exec = require('@actions/exec');
+const core = require('@actions/core');
+
+const MAX_DOMAINS = 100;
+const MAX_CERTIFICATES = 13;
+
+const gcloudOutput = async (args, bin = 'gcloud') => {
+  let output = '';
+  await exec.exec(bin, args, {
+    silent: false,
+    listeners: {
+      stdout: (data) => {
+        output += data.toString('utf8');
+      },
+    },
+  });
+  return output.trim();
+};
```

Comment:
Could this code be extracted to its own module? It looks like it is repeated in two files?

Reply:
yes, that's part of some refactoring i need to do :) 👍